prisma/schema.prisma

```diff
@@ -13,3 +13,17 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
```

Comment:
**nitpick**: Use snake_case consistently for field names. Consider renaming `short_url` to `shortUrl` and `original_url` to `originalUrl` to follow camelCase convention, which is commonly used in JavaScript and TypeScript projects.